etc/001.yml

```diff
@@ -1,4 +1,4 @@
-v: 2
```

Comment:
Does the `edge.router.yml` file also need to be updated?

Reply:
Update pushed.